verticapy/_config/config.py

```diff
@@ -148,6 +153,182 @@ def set_option(key: str, value: Any = None) -> None:
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("footer_on", False)
+        html_file = open("figures/_config_config_set_option_3.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_3.html
+
+    .. code-block:: python
+
+        # Sets the maximum number of columns to display
+        set_option("max_columns", 3)
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("max_columns", 3)
+        html_file = open("figures/_config_config_set_option_4.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_4.html
+
+    .. code-block:: python
+
+        # Sets the maximum number of row to display
+        set_option("max_rows", 5)
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("max_rows", 5)
+        html_file = open("figures/_config_config_set_option_5.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_5.html
+
+    .. code-block:: python
+
+        # Light mode
+        set_option("mode", "light")
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("mode", "light")
+        html_file = open("figures/_config_config_set_option_6.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_6.html
+
+    .. code-block:: python
+
+        # Full mode
+        set_option("mode", "full")
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("mode", "full")
+        html_file = open("figures/_config_config_set_option_7.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_7.html
+
+    .. code-block:: python
+
+        # Display the loading bar
+        set_option("percent_bar", True)
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("percent_bar", True)
+        html_file = open("figures/_config_config_set_option_8.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_8.html
+
+    .. ipython:: python
+
+        # Display queries and their execution times
+        set_option("sql_on", True)
+        set_option("time_on", True)
+        @savefig _config_config_set_option_corr.png
+        titanic.corr()
+
+    .. ipython:: python
+
+        # Hide queires and execution times
+        set_option("sql_on", False)
```

Comment:
duplicate + typo. Please move the comment out of the block whenever it is possible.